src/components/Steps.tsx

```diff
@@ -56,7 +56,7 @@ const Steps: React.FC = () => {
               </TableRow>
             </TableBody>
           </Table>
-          <p className="text-destructive mt-4 text-sm font-semibold">
+          <p className="text-red-500 mt-4 text-sm font-semibold">
```

Comment:
Don't make this red-500 again and again please. We have defined colors. Check global.css file please.

Reply:
ok